app/helpers/searches_helper.rb

```diff
@@ -26,4 +26,11 @@ def aggregation_count(aggregration, duration, buckets_pos)
     update_info = (duration == 30.days ? t("searches.show.month_update", count: count) : t("searches.show.week_update", count: count))
     link_to update_info, path, class: "t-link--black"
   end
+
+  def aggregation_yanked(yanked_gem)
+    return unless yanked_gem
+
+    path = search_path(params: { query: params[:query], yanked: true })
+    link_to t("versions.version.yanked") + " (1)", path, class: "t-link--black"
```

Comment:
Is this fine?
`t("versions.version.yanked") + " (1)"`
Or better to make a new key? Though count will always be 1

Reply:
Added a new key with i18n value